taichi/ir/frontend_ir.h

```diff
@@ -430,6 +430,39 @@ class GlobalPtrExpression : public Expression {
   }
 };
 
+class GlobalTensorElementExpression : public Expression {
+ public:
+  Expr var;
+  ExprGroup indices;
+  int cols;
+  bool is_AOS;
```

Comment:
Here and below: let's use `is_aos`.

```suggestion
  bool is_aos{false};
```

Reply:
https://github.com/taichi-dev/taichi/pull/2543/commits/46dd8a20ddde35b90cda21b34fcf06523ac99edb